docs/docs/index.md

```diff
@@ -260,6 +260,8 @@ If the first character of `[value]` is `@` the subsequent path will be read.
 * `rq enqueue path.to.func ':{"json": "abc"}'` -> `queue.enqueue(path.to.func, {'json': 'abc'})`
 * `rq enqueue path.to.func 'key:={"json": "abc"}'` -> `queue.enqueue(path.to.func, key={'json': 'abc'})`
 * `rq enqueue path.to.func '%1, 2'` -> `queue.enqueue(path.to.func, (1, 2))`
+* `rq enqueue path.to.func '%None'` -> `queue.enqueue(path.to.func, None)`
+* `rq enqueue path.to.func '%True'` -> `queue.enqueue(path.to.func, True)`
```

Comment:
Could you please add another example, along with tests for passing in a Python dict?
```
rq enqueue path.to.func key%='{"foo": True}'` -> `queue.enqueue(path.to.func, key={"foo": True})
```